docs/src/almalinux_instructions.rst

```diff
@@ -0,0 +1,257 @@
+    Available versions can be found `here <https://www.npmjs.com/package/carta-controller>`_.
+
+.. note::
+    
+    On RHEL7/CentOS7 the carta-controller package can not run with the default gcc version 4.8.5 (there would be an error due to ``node-linux-pam``). 
+    A work around is to install a newer GCC version from source in order to get a newer ``libstdc++.so.6``, then add the location of the newer 
+    ``libstdc++.so.6`` to the LD_LIBRARY_PATH. After that, carta-controller can run on RHEL7/CentOS7.
+
+
+4. Install the carta-backend component
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The easiest way may be to install the carta-backend is from our cartavis RPM repository.
+
+.. code-block:: shell
+
+    # Install carta-backend
+    sudo curl https://packages.cartavis.org/cartavis-el8.repo --output /etc/yum.repos.d/cartavis.repo
+    sudo dnf -y install 'dnf-command(config-manager)'
+    sudo dnf -y install epel-release
+    sudo dnf -y config-manager --set-enabled powertools
+    sudo dnf -y install carta-backend
+
+    # Check that the backend can run and matches the major version number of the controller
+    /usr/bin/carta_backend --version
+
+
+.. note::
+    
+    If you install the beta version of carta-controller, you need to install the beta version of the carta-backend ``sudo dnf -y install carta-backend-beta``.
+
+
+5. Install Nginx
+~~~~~~~~~~~~~~~~
+
+carta-controller requires a webserver. Here we use `NGINX <https://www.nginx.com/>`_, but Apache should work too.
+
+.. code-block:: shell
+
+    # Install nginx:
+    sudo dnf install -y nginx
+    sudo systemctl start nginx
+    sudo systemctl enable nginx
+    sudo setsebool -P httpd_can_network_connect 1
+    sudo firewall-cmd --permanent --zone=public --add-service=http
+    sudo firewall-cmd --permanent --zone=public --add-service=https
+    sudo firewall-cmd --reload
+
+    # Set up the nginx configuration file using our sample configuration file linked below:
+    sudo cd /etc/nginx/conf.d/
+    sudo vi /etc/nginx/conf.d/carta.conf
+    sudo systemctl restart nginx
+
+    # Check it is running:
+    sudo systemctl status nginx
+
+A :ref:`sample configuration file<example_nginx>` is provided in the configuration section. This should be adapted to your server configuration.
+
+.. note::
+    If there are problems, you can debug with ``journactl -xe`` and checking log files in ``/var/log/nginx/``.
+
+
+6. Create the 'carta' user
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For security, we recommend not to run the carta-controller as the root user. Therefore we create a new user called ``carta`` and make part it part of a new group called ``carta-users``. 
+We will allow any user in the ``carta-users`` group to run ``/usr/bin/carta_backend`` and the script to close the carta-backend; ``/usr/local/bin/carta-kill-script`` by adding a custom entry to the ``sudoers`` file.
```

Comment:
The carta user should not be in the carta-users group; that's for the user accounts.